NeoML/src/TraditionalML/GradientBoost.cpp

```diff
@@ -268,11 +268,12 @@ CPtr<IModel> CGradientBoost::Train( const IProblem& problem )
 		*logStream << "\nGradient boost training started:\n";
 	}
 
+	CPtr<const IProblem> notNullWeightsView = FINE_DEBUG_NEW CProblemNotNullWeightsView( &problem );
```

Comment:
`CProblemNotNullWeightsView` must be used in `TrainRegression` as well.

Reply:
unified with a common view over IMultivariateRegressionProblem